### PR TITLE
add router fallback description

### LINKED ISSUE #310: Routine Maintenance of Plugin Documents
### What would you like to be added?

Routine Maintenance of Plugin Documents

### Why is this needed?

update content of plugin document

### PR DESCRIPTION
**Pull request by daizhenyu**
What type of PR is this?

/Document

What this PR does / why we need it?

add router fallback description

Which issue(s) this PR fixes:

Fixes #310

### CODE REVIEW

**Review by provenceee — APPROVED** [review]

File: docs/zh/document/plugin/router.md [hunk 1]
```diff
@@ -126,7 +137,9 @@
               group: green                 # 满足match条件的实例路由到group标签为green的实例
   ```
   
-  **上述路由规则解释：** zone标签配置为hangzhou的消费端实例在调用下游服务时，优先调用zone标签也为hangzhou的下游实例。version标签配置为1.0.0的消费端实例在调用下游服务时，请求的20%会路由到group标签为red的服务实例，80%会路由到group标签为green的服务实例。
+  **上述路由规则解释：** zone标签配置为hangzhou的消费端实例在调用下游服务时，优先调用zone标签也为hangzhou的下游实例。version标签配置为1.0.
+  0的消费端实例在调用下游服务时，请求的20%会路由到group标签为red的服务实例，80%会路由到group标签为green的服务实例。
```
Comment (lilai23) [inline docs/zh/document/plugin/router.md]: unnecessary enter

File: docs/zh/document/plugin/router.md [hunk 2]
```diff
@@ -77,9 +81,15 @@
...
+        fallback:                        # 满足match条件，但路由的实例为空时，配置fallback策略，路由到group标签为black的实例
+          - tags:
+              group: black               # 满足fallback条件的实例路由到group标签为black的实例
+            weight: 100                  # 权重值为100%
   ```
   
-  **上述路由规则解释：** attachments信息中id属性值为1的请求20%会路由到版本号为1.0.0的服务实例，80%会路由到版本号为1.0.1的服务实例。其他请求20%会路由到group标签为red的服务实例，80%会路由到group标签为green的服务实例。
+  **上述路由规则解释：** attachments信息中id属性值为1的请求20%会路由到版本号为1.0.0的服务实例，80%会路由到版本号为1.0.
+  1的服务实例，当匹配的服务实例为空时，执行fallback策略，100%会路由到版本号为0.9.
+  0的服务实例。其他请求20%会路由到group标签为red的服务实例，80%会路由到group标签为green的服务实例，当匹配的服务实例为空时，执行fallback策略，100%会路由到group为black的服务实例。
```
Comment (lilai23) [inline docs/zh/document/plugin/router.md:93]: unnecessary enter

File: docs/zh/document/plugin/router.md [hunk 3]
```diff
@@ -77,9 +81,15 @@
...
```
Comment (lilai23) [inline docs/zh/document/plugin/router.md]: unnecessary enter

File: docs/zh/document/plugin/router.md (hunk 1 above)
Comment (daizhenyu) [reply docs/zh/document/plugin/router.md]: done

File: docs/zh/document/plugin/router.md (hunk 3 above)
Comment (daizhenyu) [reply docs/zh/document/plugin/router.md]: done

**Pull request merged**